.circleci/workflows.yml

```diff
@@ -2470,6 +2472,8 @@ jobs:
           repo: cypress-realworld-app
           browser: chrome
           wait-on: http://localhost:3000
+          # TODO: remove this once testing is complete. We are testing the binary against the vite update inside RWA that is on Node 18
+          pull_request_id: 1396
```

Comment:
What are the criteria for testing being complete?

Reply:
I need both [cypress-example-recipes](https://github.com/cypress-io/cypress-example-recipes/pull/844) and [cypress-realworld-app](https://github.com/cypress-io/cypress-realworld-app/pull/1396) PRs to be complete and merged in so they can run in the Node 18 run time. Currently they run on webpack 4 / react-scripts 4 and don't work in 18. Once those are in, I can remove these lines of code

Reply:
what I am thinking is we have a strategic release where once all 3 are approved we merge those two in and push an update here to remove these lines and likely admin merge in since we know the changeset. Or even better, if closed PRs still work, we could just create a separate PR